.ci.yaml

```diff
@@ -52,7 +52,7 @@ platform_properties:
           {"dependency": "open_jdk", "version": "version:11"}
         ]
       device_type: none
-      cpu: x86
+      cpu: arm64
```

Comment:
Once the arm tests all pass, this will be removed so the scheduler can choose either arm64 or x64 depending on capacity.